eng/Versions.props

```diff
@@ -58,7 +58,8 @@
     <MicrosoftDotNetXHarnessTestRunnersXunitVersion>1.0.0-prerelease.22256.2</MicrosoftDotNetXHarnessTestRunnersXunitVersion>
     <MicrosoftDotNetXHarnessCLIVersion>1.0.0-prerelease.22256.2</MicrosoftDotNetXHarnessCLIVersion>
     <TizenUIExtensionsVersion>0.8.1</TizenUIExtensionsVersion>
-    <SvgSkiaVersion>0.4.1</SvgSkiaVersion>
+    <SvgSkiaPackageVersion>0.5.13</SvgSkiaPackageVersion>
```

Comment:
I upgraded Svg because there was some weird hacks to make it work with the old skia. The new version works fine without hacks.